src/main/fc/runtime_config.c

```diff
@@ -34,34 +34,37 @@ static uint32_t enabledSensors = 0;
 
 // Must be no longer than OSD_WARNINGS_MAX_SIZE (11) to be displayed fully in OSD
```

Comment:
Do we need to trim names here ?

Reply:
OSD seems to use 32byte buffer, so overflow is unlikely. I did not check if OSD code does trim it ...